.env.production

```diff
@@ -2,71 +2,7 @@
 VITE_FEATURE_MIXPANEL=true
 VITE_ENABLE_ADDRESSABLE=true
 VITE_FEATURE_ADDRESS_BOOK=false
-
 VITE_FEATURE_THORCHAIN_TCY_ACTIVITY=false
 
 # mixpanel
 VITE_MIXPANEL_TOKEN=9d304465fc72224aead9e027e7c24356
-
-# unchained
-VITE_UNCHAINED_ETHEREUM_HTTP_URL=https://api.ethereum.shapeshift.com
-VITE_UNCHAINED_ETHEREUM_WS_URL=wss://api.ethereum.shapeshift.com
-VITE_UNCHAINED_AVALANCHE_HTTP_URL=https://api.avalanche.shapeshift.com
-VITE_UNCHAINED_AVALANCHE_WS_URL=wss://api.avalanche.shapeshift.com
-VITE_UNCHAINED_OPTIMISM_HTTP_URL=https://api.optimism.shapeshift.com
-VITE_UNCHAINED_OPTIMISM_WS_URL=wss://api.optimism.shapeshift.com
-VITE_UNCHAINED_BNBSMARTCHAIN_HTTP_URL=https://api.bnbsmartchain.shapeshift.com
-VITE_UNCHAINED_BNBSMARTCHAIN_WS_URL=wss://api.bnbsmartchain.shapeshift.com
-VITE_UNCHAINED_POLYGON_HTTP_URL=https://api.polygon.shapeshift.com
-VITE_UNCHAINED_POLYGON_WS_URL=wss://api.polygon.shapeshift.com
-VITE_UNCHAINED_GNOSIS_HTTP_URL=https://api.gnosis.shapeshift.com
-VITE_UNCHAINED_GNOSIS_WS_URL=wss://api.gnosis.shapeshift.com
-VITE_UNCHAINED_ARBITRUM_HTTP_URL=https://api.arbitrum.shapeshift.com
-VITE_UNCHAINED_ARBITRUM_WS_URL=wss://api.arbitrum.shapeshift.com
-VITE_UNCHAINED_ARBITRUM_NOVA_HTTP_URL=https://api.arbitrum-nova.shapeshift.com
-VITE_UNCHAINED_ARBITRUM_NOVA_WS_URL=wss://api.arbitrum-nova.shapeshift.com
-VITE_UNCHAINED_BASE_HTTP_URL=https://api.base.shapeshift.com
-VITE_UNCHAINED_BASE_WS_URL=wss://api.base.shapeshift.com
-VITE_UNCHAINED_MONAD_HTTP_URL=https://rpc.monad.xyz
-VITE_UNCHAINED_MONAD_WS_URL=wss://rpc3.monad.xyz
-VITE_UNCHAINED_HYPEREVM_HTTP_URL=https://rpc.hyperliquid.xyz/evm
```

Comment:
That was not actually used anywhere, safe to remove. We actually use `VITE_HYPEREVM_NODE_URL=https://rpc.hyperliquid.xyz/evm` which *is* in .env